### DISCUSSION TITLE
Can You Help Me

### DISCUSSION THREAD

**Discussion by autoattack999** [masterking32/MasterHamsterKombatBot#461] [General]
Anyone know this problem?
I open this on Android, instead a warning appears like this.
But after I tried to log in via the Web Telegram, hamster kombat can be opened, is there a solution?
![IMG_20240917_173917_919](https://github.com/user-attachments/assets/f3cc240c-4690-48d9-9d2c-305ce3cb3eb3)

**Comment by Fy0urM** [comment]
the application died a bit from another update made by high-class programmers
